asm-to-pil/src/romgen.rs

```diff
@@ -214,11 +215,24 @@ pub fn generate_machine_rom<T: FieldElement>(
             parse_function_statement("_loop;"),
         ])]);
 
+        let latch = instruction_flag(RETURN_NAME);
+        let last_step = "_block_enforcer_last_step";
+        let operation_id_no_change = "_operation_id_no_change";
+
         machine.pil.extend([
             // inject the operation_id
             parse_pil_statement(&format!(
                 "col witness {operation_id}(i) query (\"hint\", {sink_id})"
             )),
+            // inject last step
+            parse_pil_statement(&format!("col constant {last_step} = [0]* + [1]")),
```

Comment:
maybe we can already use the new syntax here?
```suggestion
            parse_pil_statement(&format!("let {last_step}: col = [0]* + [1]")),
```